components/dictionary.txt

```diff
@@ -341,8 +342,10 @@ scater
 SCE
 scran
 scRNA
+Sebire
 Sergushichev
 setosa
+Shern
```

Comment:
Did I sing the alphabet song to confirm that `h` really is before `i`? Yes, yes I did. It is.

Reply:
I also sang the alphabet song and do every time I add something to the dictionary.